### DISCUSSION TITLE
How to store the video streaming in core-data?

### DISCUSSION THREAD

**Discussion by vediyappan-villali** [edgexfoundry/community#73] [Q&A]
I have an use case to store the video streamed by webcam that has to be stored in EdgeX core-data. I am receiving the video stream through EdgeX device-usb-camera. How to store the data in EdgeX core-data?

I tried with `cmd/core-data/res/configuration.toml:PersistData = true`. But I could see the value of readings of the webcam event(s) is empty.

On other hand, I am able to receive the streamed data in RTSP server and store it outside EdgeX. I just want to store the video stream in EdgeX itself. How to do that?

**Comment by cloudxxx8** [comment]
Currently, the persistence of core-data is Redis, and it's not designed to store the binary data.
You can write the an App Service to store the video stream to a proper database.

**Reply by vediyappan-villali** [reply]
@cloudxxx8 Ok, thank you for the info.
